src/modules/agents/server/procedures.ts

```
@@ -0,0 +1,17 @@
+import { TRPCError } from "@trpc/server";
+
+export const agentsRouter = createTRPCRouter ({
+  getMany: baseProcedure.query(async () => {
+    const data = await db 
+      .select()
+      .from(agents);
+
+    // await new Promise((resolve) => setTimeout(resolve, 5000));
+    throw new TRPCError({ code: "BAD_REQUEST"});
+
+    return data;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)